labstep/resourceItem.py

```diff
@@ -103,7 +103,7 @@ def newResourceItem(user, resource, name=None, availability=None,
 
 
 def editResourceItem(resourceItem, name=None, availability=None,
-                     quantity_amount=None, quantity_unit=None,
+                     number=None, unit=None,
```

Comment:
Sorry for resource_item the backend does still use `quantity_amount` and `quantity_unit`!

Reply:
Good catch!